content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -129,20 +168,29 @@ the :guilabel:`Reverse Transfer` pop-up window.
    :align: center
```

Comment:
Only one ticket in two runbots had the return button visible- all the others didn't. Not sure what the parameters are, but maybe mention that? I didn't even see a product on the ticket- but then I saw it on the return pop-up, so I couldn't really verify by looking at the ticket if the return button should have been there.

Reply:
There is an Important admonition block on line 171 that mentions this. The Return button will only appear if there is a recorded delivery in the database for this customer.